crates/turborepo-paths/src/relative_unix_path.rs

```diff
@@ -4,10 +4,11 @@ use std::{
 };
 
 use camino::{Utf8Path, Utf8PathBuf};
+use serde::Serialize;
 
 use crate::{AnchoredSystemPathBuf, PathError, RelativeUnixPathBuf};
 
-#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize)]
```

Comment:
This implementation is no longer necessary so we should drop it.

Reply:
totally: fixed in e5ae321c90af89d648d465713ff6ee2c52fc2534